### PR TITLE
Use as default date today and as default city NY

### PR DESCRIPTION
**Pull request by JaapHaitsma**
My first PR ever for a React project. Not sure it is useful as with material UI you probably have many changes. Just wanted to get some practice for React.

If useful commit if not you can just delete. I'm just happy that I made something work in React

### CODE REVIEW

File: src/pages/LandingPage.tsx [hunk 1]
```diff
@@ -25,14 +25,22 @@ const NoShows = styled.div`
   top: 1rem;
 `;
 
+const getDateString = () => {
```
Comment (mzielonkowski-merapar) [inline src/pages/LandingPage.tsx:28]: we have ./src/utils/getTodayDateISO.ts , please use that function getTodayDateISO() to get ISO date

File: src/pages/LandingPage.tsx [hunk 2]
```diff
@@ -25,14 +25,22 @@ const NoShows = styled.div`
...
 export const LandingPage = () => {
   const apiKey = process.env.REACT_APP_API_KEY as string;
   const baseUrl = process.env.REACT_APP_BASE_URL as string;
   const moviesTheatrePathName = process.env
     .REACT_APP_MOVIES_THEATRE_PATH_NAME as string;
 
-  const [selectedDate, setSelectedDate] = useState("");
-  const [zipcode, setZipcode] = useState(0);
+  const [selectedDate, setSelectedDate] = useState(getDateString());
+  const [zipcode, setZipcode] = useState(11554);
```
Comment (mzielonkowski-merapar) [inline src/pages/LandingPage.tsx:43]: this 11554 is so called magic number, please use const for it to keep it with nice name (as seen in the PR's name it is NYC one of zipcodes)

**Review by mzielonkowski-merapar — CHANGES_REQUESTED** [review]
please change code with my comments

**Comment by JaapHaitsma** [comment]
Sorry @mzielonkowski-merapar I missed your comments. Agree with them but as you guys will move the code to material UI datepicker it does not make that much sense to still merge this

**Pull request closed without merging**